src/components/imports/DropboxChooser.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import loadScript from 'load-script';
+
+const DROPBOX_SDK_URL = 'https://www.dropbox.com/static/api/2/dropins.js';
+const SCRIPT_ID = 'dropboxjs';
+
+let scriptLoadingStarted = false;
```

Comment:
Make this a part of the DropboxChooser class state instead. This would make it possible to use it for showing a loader in future if the need arises.